lib/embed/media_tag.rb

```diff
@@ -4,6 +4,7 @@ module Embed
   # and HLS is used as a <source> within the <video> or <audio> tag.
   class MediaTag
     SUPPORTED_MEDIA_TYPES = [:audio, :video].freeze
+    MEDIA_INDEX_CONTROL_HEIGHT = 24
```

Comment:
naive question:   is it a coincidence that the scss thing is 24px and this constant is 24?  I'm assuming  "no" but there is no good way to share a constant between ruby code and scss.